### PR TITLE
fix imports firebase

### PR DESCRIPTION
**Pull request by avelow**
fix issue "It looks like you're using the development build of the Firebase JS SDK" #33

### CODE REVIEW

File: projects/akita-ng-fire/src/lib/collection/collection.service.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ import {
   getEntityType,
   getIDType,
 } from '@datorama/akita';
-import { firestore } from 'firebase';
+import * as firebase from 'firebase/app';
```
Comment (GrandSchtroumpf) [inline projects/akita-ng-fire/src/lib/collection/collection.service.ts]: Same as above

File: projects/akita-ng-fire/src/lib/utils/types.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { AngularFirestore, DocumentChangeAction } from '@angular/fire/firestore';
 import { EntityStore, EntityState, getEntityType } from '@datorama/akita';
 import { Observable } from 'rxjs';
-import { firestore } from 'firebase';
+import * as firebase from 'firebase/app';
```
Comment (GrandSchtroumpf) [inline projects/akita-ng-fire/src/lib/utils/types.ts]: Same as above

File: projects/akita-ng-fire/src/lib/auth/auth.service.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
 import { inject } from '@angular/core';
 import { AngularFireAuth } from '@angular/fire/auth';
 import { AngularFirestore, AngularFirestoreCollection } from '@angular/fire/firestore';
-import { auth, User } from 'firebase';
+import * as firebase from 'firebase/app';
```
Comment (GrandSchtroumpf) [inline projects/akita-ng-fire/src/lib/auth/auth.service.ts]: Please import only what you need : `import { auth, User } from 'firebase/app';`

**Comment by avelow** [comment]
This is the way firebase recommand to import.

I got this in my console while using your library :
```
It looks like you're using the development build of the Firebase JS SDK.
When deploying Firebase apps to production, it is advisable to only import
the individual SDK components you intend to use.

For the module builds, these are available in the following manner
(replace <PACKAGE> with the name of a component - i.e. auth, database, etc):

CommonJS Modules:
const firebase = require('firebase/app');
require('firebase/<PACKAGE>');

ES Modules:
import firebase from 'firebase/app';
import 'firebase/<PACKAGE>';

Typescript:
import * as firebase from 'firebase/app';
import 'firebase/<PACKAGE>';
```

**Comment by GrandSchtroumpf** [comment]
Yes, I understand that `import { firestore } from 'firebase';` displays this message. 
But I don't understand that `import { firestore } from 'firebase/app';` also displays this message if `import * as firebase from 'firebase/app';` doesn't display it.
It's just spreading the exported values, there is no reason to change the behaviour.

**Comment by avelow** [comment]
Changes done

**Pull request merged**